README.md

```diff
@@ -45,7 +45,7 @@ OpenSearch Security is an OpenSearch plugin that offers encryption, authenticati
 
 ## Documentation
 
-Please refer to the [technical documentation](https://opendistro.github.io/for-elasticsearch-docs/docs/security/configuration/) for detailed information on installing and configuring opensearch-security plugin.
+Please refer to the [technical documentation](https://docs-beta.opensearch.org/docs/security/configuration/) for detailed information on installing and configuring opensearch-security plugin.
```

Comment:
This should be the main page of our plugin

https://docs-beta.opensearch.org/docs/security/

Reply:
@hsiang9431-amzn The main page that you refer to doesn't look explain: "for detailed information on installing and configuring opensearch-security plugin". Would it be better to remain it the page that is equivalent to [the original page](https://opendistro.github.io/for-elasticsearch-docs/docs/security/configuration/) that the old link redirects to?